plugin/src/main/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfigurator.java

```diff
@@ -18,6 +22,10 @@
 @Extension
 @Restricted(NoExternalUse.class)
 public class HudsonPrivateSecurityRealmConfigurator extends DataBoundConfigurator {
+    private static final Logger logger = LoggerFactory.getLogger(HudsonPrivateSecurityRealmConfigurator.class);
+    /// matches HudsonPrivateSecurityRealm.JBCRYPT_HEADER
+    private static final String HASHED_PASSWORD_PREFIX = "#jbcrypt:";
+    private static final VersionNumber MIN_VERSION_FOR_HASHED_PASSWORDS = new VersionNumber("2.161");
```

Comment:
Is this code needed? The baseline for this plugin requires this version

Reply:
So this was me trying to fix the CI system:

https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fconfiguration-as-code-plugin/detail/PR-683/10/pipeline/10

It seems that you have tests which run against the version specified (although judging from the console output, it's still running against 2.107.1:

```
:\Jenkins\workspace\figuration-as-code-plugin_PR-683>mvn --batch-mode --show-version --errors -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -s C:\Jenkins\workspace\figuration-as-code-plugin_PR-683@tmp/settings-azure.xml --update-snapshots -Dmaven.repo.local=C:\Jenkins\workspace\figuration-as-code-plugin_PR-683@tmp/m2repo -Dmaven.test.failure.ignore -Dset.changelist -Djenkins.version=2.107.1 -Daccess-modifier-checker.failOnError=false clean install 
```

I can't seem to figure out where the build is pulling this older version. I'm happy to remove this workaround, but it also seems like a reasonable way to support a form of backwards compatibility.

Reply:
it'll be because you aren't a trusted collaborator, it will use the master Jenkinsfile even if you change it in the PR


Not sure on whether the double guard is needed, @casz is it common practice to add version guards in the code and a minimum version in the plugin?